package.json

```diff
@@ -12,7 +12,7 @@
     "backend"
   ],
   "engines": {
-    "node": ">= v18.17.0"
+    "node": ">= v20.13.1"
```

Comment:
Was there any updates on the `yarn.lock`

Reply:
It seems like all packages where compatible with both `node` versions, so `yarn` did not had to update any packages.